scripts/scr_weapon/scr_weapon.gml

```diff
@@ -753,4 +750,3 @@ function scr_weapon(argument0, argument1, argument2, argument3, argument4, argum
 
 
 }
-}
```

Comment:
this } was supposed to be at line 646. The whole situation was caused by unresolved merge conflicts in PR #12